src/pages/gradient-text.ts

```diff
@@ -214,3 +216,16 @@ function getValues() {
 }
 resetValues();
 getValues();
+
+// Tailwind codecopy handler
+function tailwindHandler() {
+  getTailwindCode.addEventListener('click', () => {
+    showPopup(
+      'Tailwind Code Copied',
+      'Code has been successfully copied to clipboard',
+      'success'
+    );
+  });
+}
+
+tailwindHandler();
```

Comment:
not reusable code, we already have one here, that's what i meant by track where the CSS code copy was for, which meant create a similar function called copyTailwindCodeToClipboard, then rename this one to copyCSSCodeToClipboard. call the act on css generator(which you will rename too and create a new function for the tailwind with the switch statement without the code under it.
https://github.com/Dun-sin/Code-Magic/blob/6ec52511508cde36b3c9cf96a5cc991aef0d1d5b/src/lib/packages.ts#L67

Here's the act on generator function:
https://github.com/Dun-sin/Code-Magic/blob/6ec52511508cde36b3c9cf96a5cc991aef0d1d5b/src/lib/packages.ts#L95-L251

Reply:
in other words duplicate the actongenerator function change the name and make the codeToCopy instances to be an empty string for all cases.